snitun/multiplexer/message.py

```diff
@@ -2,6 +2,7 @@
 
 import binascii
 import os
+from typing import NamedTuple
 
 import attr
```

Comment:
Note that `attr` can be dropped after optimizations merge and the remaining usage is converted to a dataclass